routes/routes.go

```diff
@@ -0,0 +1,55 @@
+package routes
+
```

Comment:
This is a great abstraction to move this code from your main.go into a separate module. If I were doing a full-scale production system, I would have these routing methods call other methods still that actually did the crud operations.